app/code/Magento/AdminAdobeIms/App/Action/Plugin/Authentication.php

```diff
@@ -56,4 +56,9 @@ public function __construct(
 
         $this->_openActions[] = ImsCallback::ACTION_NAME;
     }
+
+    private function unusedMethod()
```

Comment:
What is the purpose of this method? Left over debug code?

Reply:
Can be seen so, yes. I want to execute PHPMD only on the changed files within a PR to speed up the process. To test this behavior, I need a PHPMD error :). Can be removed as soon as I've seen the result and the nightly builds run again.